pkg/standalone/uninstall.go

```diff
@@ -62,6 +70,11 @@ func Uninstall(uninstallAll bool, dockerNetwork string) error {
 		fmt.Println("WARNING: could not delete run data file")
 	}
 
+	err = removeDefaultComponentsFolder()
+	if err != nil {
+		fmt.Println("WARNING: could not delete default components folder")
```

Comment:
probably print location of the default components folder here?

Reply:
done